### ISSUE TITLE
Needed feature additions request

### ISSUE THREAD

**Issue by 1red-ryu** [FellowHead/flpjck#6]
This is a good tool but we need the FL export settings menu , basically:

-Choose mp3 quality
-Split mixer tracks
-Enable insert effects
-Enable master effects
-Full song / pattern mode 
 etc.

**Comment by doodle-alt** [comment]
I agree, these options would be nice to have. Unfortunately, the FL Studio command line interface doesn't support any parameters except for output format. You can read about it in the [manual](https://www.image-line.com/fl-studio-learning/fl-studio-online-manual/html/fformats_save_export.htm), under the last page section titled "Command line export options".

A workaround I've tried during development was to not use the CLI at all and instead simulate keystrokes inside FL. In theory, your issue is resolveable, but it would need a completely different approach and lots of time. Maybe I'll get to this in the future.

**Comment by 1red-ryu** [comment]
Would be great thanks for the reply and I'll be looking forward to updates
/ telling people about this when I can
